README.md

```diff
@@ -1,2 +1,120 @@
 # neofs-sdk-go
-Go implementation of NeoFS SDK
+Go implementation of NeoFS SDK. It contains high-level version-independent wrappers
+for structures from [neofs-api-go](https://github.com/nspcc-dev/neofs-api-go) as well as
+helper functions for simplifying node/dApp implementations.
+
+## Repository structure
```

Comment:
Some packages are missing, like `audit`, `object`, etc. Probably it is intentional, so I propose at least note them somehow. For example group it as `packages with API message definitions: accounting, acl, checksum, etc.`